docs/source/endpoints/userschema.md

```diff
@@ -37,3 +37,22 @@ The server will respond with the user schema.
 The user schema uses the same serialization as the type's JSON schema.
 
 See {ref}`types-schema` for detailed documentation about the available field types.
+
+## Getting the registration form
+
+In Plone we can configure each of the fields of the user schema to be available only in the user edit form, in the registration form or in both of them.
```

Comment:
I think this is your intent, but please verify.

```suggestion
In Plone you can configure each of the fields of the user schema to be available in only one of either the user edit form or registration form, or in both of them.
```

Reply:
that's right.